addons/project_timesheet_holidays/models/account_analytic.py

```diff
@@ -34,3 +34,9 @@ def write(self, vals):
         if not self._check_can_update_timesheet():
             raise UserError(_('You cannot modify timesheets that are linked to time off requests. Please use the Time Off application to modify your time off requests instead.'))
         return super().write(vals)
+
+    def _get_favorite_project_id_domain(self, employee_id=False):
+        return expression.AND([
+            super()._get_favorite_project_id_domain(employee_id),
+            [('holiday_id', '=', False), ('global_leave_id', '=', False)],
+        ])
```

Comment:
Need a new blank to make the runbot happy. 🙂 

```suggestion
        ])

```